src/main/java/org/openmainframeproject/cobolcheck/features/interpreter/InterpreterController.java

```diff
@@ -502,4 +506,30 @@ private void resetPossibleMockValues(){
         possibleMockIdentifier = null;
         possibleMockType = null;
     }
+
+    public List<String> getSectionLines(){
```

Comment:
I think the word 'section' in all of these methods can be a bit misleading, as it is covering both sections and paragraphs. Either they should be called `...SectionOrParagraph...`, which is not that pretty, or we can check if IBM has a word that encompasses both paragraph and section 😄